### ISSUE TITLE
[Python] Exception launched by launch_vessel_from_vab

### ISSUE THREAD

**Issue by kamaradclimber** [krpc/krpc#599]
- KSP version: 1.11.0.3045
 - KRPC version: 0.4.8

When using the following script:
```
#!/usr/bin/env python

import krpc
conn = krpc.connect(name='Hello World')
vessel_name = conn.space_center.launchable_vessels("VAB")[0]

conn.space_center.launch_vessel_from_vab(vessel_name, True)
```

The following exception is thrown:
```
Traceback (most recent call last):
  File "/ssd/home/grego/kerbal-rpc/demo_python.py", line 7, in <module>
    conn.space_center.launch_vessel_from_vab(vessel_name, True)
  File "<string>", line 1, in <lambda>
  File "/home/grego/.local/lib/python3.9/site-packages/krpc/client.py", line 159, in _invoke
    raise self._build_error(response.error)
krpc.error.RPCError: Instance constructor requires a target
Server stack trace:
  at System.Reflection.MonoCMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0000b] in <ad04dee02e7e4a85a1299c7ee81c79f6>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <ad04dee02e7e4a85a1299c7ee81c79f6>:0 
  at KRPC.Utils.Compatibility.NewLaunchSiteClear (System.String launchSite, Game game) [0x00052] in <33dee48dce9d40bcb6cf1726928e99b4>:0 
  at KRPC.SpaceCenter.Services.SpaceCenter.WaitForVesselPreFlightComplete (KRPC.SpaceCenter.Services.SpaceCenter+LaunchConfig config) [0x00016] in <f3e476445b794afca74822afa2c12a0e>:0 
  at KRPC.Continuations.ParameterizedContinuationVoid`1[TArg0].Run () [0x00000] in <33dee48dce9d40bcb6cf1726928e99b4>:0 
  at KRPC.Continuations.Continuation.RunUntyped () [0x00000] in <33dee48dce9d40bcb6cf1726928e99b4>:0 
  at KRPC.Service.Services.ExecuteCall (KRPC.Service.Scanner.ProcedureSignature procedure, KRPC.Continuations.IContinuation continuation) [0x00000] in <33dee48dce9d40bcb6cf1726928e99b4>:0 
```

The error appears to be language agnostic (I've originally ran on the error using Ruby krpc gem).

**Comment by kamaradclimber** [comment]
https://github.com/krpc/krpc/commit/845edcd05af230ef655df3030f2ed7d98d01c3d2 might be relevant in the unreleased version. I'll see how I could try this.

**Comment by SamRapier** [comment]
Is there a fix for this? I'm getting this same error.

**Comment by djungelorm** [comment]
@SamRapier Could you try with v0.5.0 (was just released) to see if this is fixed?

**Comment by djungelorm** [comment]
Confirmed still broken in latest version. Fixed by #713 and will be included in 0.5.2 release
